src/mlpack/methods/kmeans/kill_empty_clusters.hpp

```diff
@@ -51,13 +51,18 @@ class KillEmptyClusters
       const size_t emptyCluster,
       const arma::mat& /* oldCentroids */,
       arma::mat& newCentroids,
-      arma::Col<size_t>& /* clusterCounts */,
+      arma::Col<size_t>& counts /* clusterCounts */,
       MetricType& /* metric */,
       const size_t /* iteration */)
   {
-    // Kill the empty cluster.
-    newCentroids.col(emptyCluster).fill(DBL_MAX);
-    return 0; // No points were changed.
+    // Remove the empty cluster.
+      if (emptyCluster < newCentroids.n_cols)
```

Comment:
Looks like there are two extra spaces.

Reply:
It's all because of my IDE. I'm sorry for these small mistakes